aten/src/TH/generic/THTensor.h

```diff
@@ -103,6 +103,7 @@ TH_API void THTensor_(setStorage4d)(THTensor *self, THStorage *storage_, ptrdiff
 TH_API void THTensor_(narrow)(THTensor *self, THTensor *src, int dimension_, int64_t firstIndex_, int64_t size_);
 TH_API void THTensor_(select)(THTensor *self, THTensor *src, int dimension_, int64_t sliceIndex_);
 TH_API void THTensor_(transpose)(THTensor *self, THTensor *src, int dimension1_, int dimension2_);
+TH_API int THTensor_(isTransposed)(const THTensor *self);
```

Comment:
why is this added?

Reply:
It was used in MoreMath across the boundary, and I didn't think hard about how to get this function across the boundary. So into the header it went.